codegenerator/cli/templates/dynamic/init_templates/shared/package.json.hbs

```diff
@@ -41,15 +41,12 @@
     "ts-mocha": "^10.0.0",
     "ts-node": "10.9.1",
     "typescript": "5.2.2",
+    "chai": "4.3.10",
   {{/if}}    
     "mocha": "10.2.0"
   },
   "dependencies": {
-    {{#if is_typescript}}
-    "chai": "4.3.10",
```

Comment:
Moved chai to ts dev dependencies

Reply:
I don't actually think we should necessarily ship with chai. The template assertions should be simple enough to just use built in node js assert lib.

Reply:
Agree with this. We should create an issue to update the templates